### PR TITLE
hotfix 7.1.2 umiworkflow small fixes

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

Small add-on changes to the umiworkflow.

Fixed:
- Set  umi_workflow: true for panel  else false in config json
- Remove duplicate umiworkflow attribute in models.py
- Rename the umiconsensus bam headers from {samplenames} to TUMOR/NORMAL.

### CODE REVIEW

File: BALSAMIC/utils/models.py [hunk 1]
```diff
@@ -352,7 +351,6 @@ class BalsamicConfigModel(BaseModel):
     bioinfo_tools: dict
     bioinfo_tools_version: dict
     panel: Optional[PanelModel]
-    umiworkflow: bool = True
```
Comment (hassanfa) [inline BALSAMIC/utils/models.py:355]: Lets keep this here, but remove default value True.

```
umiworkflow: bool
```

**Review by hassanfa — COMMENTED** [review]
Can you please also update the Changelog? We can make a release directly after this.

File: BALSAMIC/utils/models.py (hunk 1 above)
Comment (ashwini06) [reply BALSAMIC/utils/models.py:355]: This attribute was added twice by mistake. In line 210: umiworkflow exists, so removing it from line 355

**Review by hassanfa — APPROVED** [review]

**Review by hassanfa — APPROVED** [review]
👍🏽 Looks good. Can you please run Tumor-only and Tumor-Normal and post job count here? To make sure we have correct job count and workflow structure looks good.

I'm curious to see if `*umi_consensusfiltered.merged.bam` is fine and it is not being run for more than once.

**Comment by ashwini06** [comment]
@hassanfa : Please have a look on the screenshots ⬇️ 

**Panel - TN case:**  

```balsamic run analysis -s /home/proj/long-term-stage/cancer/UMI_analysis_runs_APJ/SeraCare_207903/stableraven/stableraven.json --account=development```

Screenshot 2021-03-09 at 15 45 44

**Panel - T-only:** 

```balsamic run analysis -s /home/proj/long-term-stage/cancer/UMI_analysis_runs_APJ/SeraCare_207903/equalbug2/equalbug2.json --account=development --snakemake-opt "-F"```

Screenshot 2021-03-09 at 15 50 06

**Comment by hassanfa** [comment]
Looks good. Merge and release then. Readthedocs is complaining about missing requirements.txt, I'll make a PR after merging this to fix it.

**Pull request merged**